playbooks/gce/openshift-cluster/launch.yml

```diff
@@ -28,33 +28,33 @@
       type: "{{ k8s_type }}"
       g_sub_host_type: "{{ sub_host_type }}"
 
-  - include: ../../common/openshift-cluster/set_node_launch_facts_tasks.yml
-    vars:
-      type: "infra"
-      count: "{{ num_infra }}"
-  - include: tasks/launch_instances.yml
-    vars:
-      instances: "{{ infra_names }}"
-      cluster: "{{ cluster_id }}"
-      type: "{{ k8s_type }}"
-      g_sub_host_type: "{{ sub_host_type }}"
-
-  - set_fact:
-      a_infra: "{{ infra_names[0] }}"
-  - add_host: name={{ a_infra }} groups=service_master
-
+#  - include: ../../common/openshift-cluster/set_infra_launch_facts_tasks.yml
+#    vars:
+#      type: "infra"
+#      count: "{{ num_infra }}"
+#  - include: tasks/launch_instances.yml
+#    vars:
+#      instances: "{{ infra_names }}"
+#      cluster: "{{ cluster_id }}"
+#      type: "{{ k8s_type }}"
+#      g_sub_host_type: "{{ sub_host_type }}"
+#
+#  - set_fact:
+#      a_infra: "{{ infra_names[0] }}"
+#  - add_host: name={{ a_infra }} groups=service_master
+#
```

Comment:
Ah, I see. I just tried and I was able to launch by making the infra section look the same on gce as it does in aws.

So, it now looks like this in my branch:
```
  - include: ../../common/openshift-cluster/set_node_launch_facts_tasks.yml
    vars:
      type: "infra"
      count: "{{ num_infra }}"
  - include: tasks/launch_instances.yml
    vars:
      instances: "{{ node_names }}"
      cluster: "{{ cluster_id }}"
      type: "{{ k8s_type }}"
      g_sub_host_type: "{{ sub_host_type }}"

  - add_host:
      name: "{{ master_names.0 }}"
      groups: service_master
    when: master_names is defined and master_names.0 is defined
```

Locally, I also removed this file as it's no longer being used:
```
playbooks/common/openshift-cluster/set_infra_launch_facts_tasks.yml
```

I'm ok if we get this as a separate PR as I have some other things I had to patch to be able to launch a cluster in GCE. I'll @ mention you when I create my PR to make sure it works for you too.